### ISSUE TITLE
Efficient ways for making proxies trustful

### ISSUE THREAD

**Issue by tomislavmamic** [bridgedao/org#1]
# Efficient ways for making proxies trustful
*TL;DR* 
Staking a high amount of collateral makes sense for proxies (nodes) which wish to operate a long-term business. This resembles a lot a payment channel such as in Lightning network. Nonetheless, it might be an overkill for one-off transactions/proxies. In one-off interactions, anyone can claim a predetermined amount (transaction+fee) of money from DAO is they meet criteria - sending that amount to a certain party. In this case, they don't need to put in a deposit.

## Introduction
We have already described how collateralized proxies can be trusted to perform fiat transactions on behalf of DAO. Here, we look at other possible arrangements which enable additional use cases or decrease the costs for participants. 

If we require nodes to put a collateral into DAO to start transacting on its behalf, we put a significant burden on them. They need to own a significant amount of Ether and lock it into a smart contract. This collateral is out of their hands continuously, while they might not transact (earn) continuously. To maximize the value of their collateral, they need to make as many transactions as possible.

But, there are cases when this is not needed.

## One-off sending fiat
DAO wants to send fiat to Alice. Bob has fiat and is willing to facilitate the transaction for a fee. DAO can deposit the amount of transaction + fee in a smart time-locked contract with the ability for anyone to claim this deposit if:
a) Alice confirms they have sent her fiat
b) they prove they have sent Alice fiat

Bob will see the deposit, pay Alice fiat and claim the deposit and fee. This resembles bounties, where the person closest to Alice will be able to perform the action fastest and cheapest.

Further, the additional deposit can be demanded by both Bob and DAO in order to make them execute their transactions fast and to disincentivize cheating.

## One-off receiving fiat
DAO wants to enable Alice to pay fiat for its services. DAO will put an equivalent deposit into a smart contract. Bob can then pay the equivalent amount into the same smart contract and request a fiat payment from Alice. When Alice sends fiat, and they agree, DAO can withdraw cryptocurrency from the smart contract. 

## Keeping fiat
This is where we need "long-term" collateral deposited.
